packages/vscode-extension/package.json

```diff
@@ -648,7 +649,9 @@
     "vscode-js-debug": "file:../vscode-js-debug",
     "vscode-test": "^1.5.0",
     "ws": "^8.14.2",
-    "xml2js": "^0.6.2",
-    "tar": "^7.4.3"
+    "xml2js": "^0.6.2"
+  },
+  "dependencies": {
+    "react-scan": "https://github.com/software-mansion-labs/react-scan/releases/download/0.1.3-radon/react-scan-0.1.3-radon.tgz"
```

Comment:
we use dev dependencies for everything that is required in build-time.

Reply:
I'm not fully familiar with how the extension is built and how it accesses its dependencies yet.
Since the `react-scan` package is used _by the extension_ at runtime, it made more sense to me to include it as a regular dependency.